### PR TITLE
add memo chaininfo

### PR DESCRIPTION
**Pull request by memolabsio**
add memo chaininfo

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
I can resolve bafkreig52paynhccs4o5ew6f7mk3xoqu2bqtitmfvlgnwarh2pm33gbdrq but not QmNrdTWjkDuBSME5RZmCweVd4a34NvmSLwpW4Ca1NWwhpL

**Comment by memolabsio** [comment]
update chaininfo delete 

> I can resolve bafkreig52paynhccs4o5ew6f7mk3xoqu2bqtitmfvlgnwarh2pm33gbdrq but not QmNrdTWjkDuBSME5RZmCweVd4a34NvmSLwpW4Ca1NWwhpL
update memo chain info
delete abbas.json

File: _data/chains/eip155-1985.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "MEMO",
+    "decimals": 18
+  },
+  "infoURL": "www.memolabs.org",
+  "shortName": " memochain",
+  "chainId": 1985,
+  "networkId": 1985,
+    "explorers": [{
+    "name": "Memo Testnet Explorer",
+    "url": "https://testscan.metamemo.one:8080/",
```
Comment (ligi) [inline _data/chains/eip155-1985.json:20]:
```suggestion
    "url": "https://testscan.metamemo.one:8080",
```

File: _data/chains/eip155-985.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "MEMO",
+    "decimals": 18
+  },
+  "infoURL": "www.memolabs.org",
+  "shortName": "memochain",
+  "chainId": 985,
+  "networkId": 985,
+    "explorers": [{
+    "name": "Memo Mainnet Explorer",
+    "url": "https://scan.metamemo.one:8080/",
```
Comment (ligi) [inline _data/chains/eip155-985.json:21]:
```suggestion
    "url": "https://scan.metamemo.one:8080",
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-1985.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "https://testchain.metamemo.one:24180/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": " Memo",
+    "symbol": "MEMO",
+    "decimals": 18
+  },
+  "infoURL": "www.memolabs.org",
+  "shortName": " memochain",
```
Comment (ligi) [inline _data/chains/eip155-1985.json:15]:
```suggestion
  "shortName": "memochain",
```

Comment (ligi) [inline _data/chains/eip155-1985.json:15]:
```suggestion
  "shortName": "memochain-testnet",
```

**Pull request closed without merging**